### PR TITLE
log improvements

### PR DESCRIPTION
**Pull request by johnugeorge**
Minor log improvements

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pytorch-operator/57)

### CODE REVIEW

File: vendor/github.com/kubeflow/tf-operator/pkg/controller.v2/jobcontroller/pod.go [hunk 1]
```diff
@@ -33,7 +33,10 @@ func (jc *JobController) AddPod(obj interface{}) {
 		logger := jclogger.LoggerForPod(pod, jc.Controller.GetAPIGroupVersionKind().Kind)
 
 		if job == nil {
-			logger.Info("This pod's job does not exist")
+			// If this is a TFJob pod
```
Comment (gaocegege) [inline vendor/github.com/kubeflow/tf-operator/pkg/controller.v2/jobcontroller/pod.go]: Is it updated manually or using dep?

Comment (johnugeorge) [reply vendor/github.com/kubeflow/tf-operator/pkg/controller.v2/jobcontroller/pod.go]: it was updated via dep

Comment (johnugeorge) [reply vendor/github.com/kubeflow/tf-operator/pkg/controller.v2/jobcontroller/pod.go]: dep ensure -update

**Pull request merged**